samples/ControlCatalog.NetCore/ControlCatalog.NetCore.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
I think changing our projects to target .Net Core 3.1 should be a separate PR. We also might need to consider targeting .Net Standard 2.1 in our libraries.

Reply:
Yeah, this is just a scratch/test PR (not gonna be merged as-is) so please expect the mess :) And control catalog is not working on my system without changing it to 3.1